src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -140,4 +140,39 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     val err = service.addFileMetadata(AddFileMetadataInput("SomethingElse", fileId, "checksum"), UUID.randomUUID()).failed.futureValue
     err.getMessage should equal("SomethingElse found. We are only expecting checksum updates for now")
   }
+
+  "getFileMetadata" should "call the repository with the correct arguments" in {
+    val fileMetadataRepositoryMock = mock[FileMetadataRepository]
+    val consignmentIdCaptor: ArgumentCaptor[UUID] = ArgumentCaptor.forClass(classOf[UUID])
+    val consignmentId = UUID.randomUUID()
+    val mockResponse = Future(Seq())
+
+    when(fileMetadataRepositoryMock.getFileMetadata(consignmentIdCaptor.capture())).thenReturn(mockResponse)
+
+    val service = new FileMetadataService(fileMetadataRepositoryMock, FixedTimeSource, new FixedUUIDSource())
+    service.getFileMetadata(consignmentId).futureValue
+    consignmentIdCaptor.getValue should equal(consignmentId)
+  }
+
+  "getFileMetadata" should "return the correct values for multiple files" in {
```

Comment:
Same comment - it makes the test easier to read if you can be more specific than "correct".

Reply:
I've put `return mutiple map entries for multiple files` which I guess is more descriptive.